README.md

```diff
@@ -83,6 +83,13 @@ spec:
               # Timeout for requests to the PDNS api server
               # (in seconds)
               timeout: 30
+
+              # If the server is only allowed to edit certain zones; the
+              # default is an empty list, allowing everything.
+              allowed-zones:
```

Comment:
For discussion: Should we have the config be without the trailing `.`, and then we append it later in comparisons? It feels a little unnatural to have to put it when most places don't require it (neither do the zone names in PowerDNS).

Reply:
AFAIK, Cert-manager is quite adamant to work with fully qualified zones (I dunno about PowerDNS; I'm not the one operating ours...)

For now I've added a note in the README to be mindful about including them.